daprdocs/content/en/js-sdk-docs/js-client/_index.md

```diff
@@ -294,14 +304,114 @@ async function start() {
   const topic = "topic-a";
 
   // Configure Subscriber for a Topic
-  await server.pubsub.subscribe(pubSubName, topic, async (data: any) => console.log(`Got Data: ${JSON.stringify(data)}`));
+  // Method 1: Direct subscribtion through the `subscribe` method
+  await server.pubsub.subscribe(pubSubName, topic, async (data: any) => console.log(`Received Data: ${JSON.stringify(data)}`));
 
+  // Method 2: Direct susbcription with options through the `subscribeWithOptions` method
+  await server.pubsub.subscribeWithOptions(pubSubName, topic, { 
+    callback: async (data: any) => console.log(`Received Data: ${JSON.stringify(data)}`) 
```

Comment:
Should we add all supported options here?

Reply:
I wouldn't as this creates too much overhead

Reply:
Is there a reference for all supported options though? Or they need to check the code?